CHANGES_NEXT_RELEASE

```diff
@@ -1,2 +1,3 @@
 - Fix: The option '-multiservice' was ignored and the broker ran always in multiservice mode. Now the option is taken taken into account (Issue #725)
 - Fix: Forwarded requests always in XML (Issue #703)
+- Fix: internal substitution of servicePath defaults ("/" for update-like operations, "/#" for query-like operations)
```

Comment:
At the end, I think is good idea to have the entry. I doesn't hurt and may be useful to explain further problems related with this.

NTC